tests/functional/deprecations/01-cylc8-basic/flow.cylc

```diff
@@ -41,6 +41,11 @@
         extra log files =
         [[[job]]]
             shell = fish
+            execution polling intervals =
+            execution retry delays =
+            execution time limit =
+            submission polling intervals =
+            submission retry delays =
```

Comment:
I thought these were wrong, until I realized it was a test for the deprecations :smile:

Reply:
They're right, but only because they're wrong!